src/fal/cli/args.py

```diff
@@ -73,6 +73,13 @@ def _build_dbt_common_options(parser: argparse.ArgumentParser):
         dest=_flag_level("profiles_dir"),
     )
 
+    parser.add_argument(
```

Comment:
why did we add this to the common options but not the state and thread?

Reply:
I guess state could have gone here.